hw6_social.py

```diff
@@ -54,7 +61,11 @@ def parsePosition(fromString):
 Returns: str
 '''
 def parseState(fromString):
-    return
+    start=fromString.find(" from ")+len(" from ")
+    fromString=fromString[start:]
+    end=fromString.find(')")')

```

Comment:
In line 66, the string which you are finding is incorrect. Please check it once again.

Reply:
Rectified the error mam.